File: RedSheet.DbEntities/Models/vRapportPlan.cs

using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using Rx.Core.Data.Attributes;
using RedSheet.DbEntities.Enums;

namespace RedSheet.DbEntities.Models
{
	[Table("vRapportPlans", Schema = "dbo")]
	[Serializable]
	public partial class vRapportPlan  
	{
 
	[DatabaseGenerated(DatabaseGeneratedOption.None)] 
	[Key]
	public int CultureCountryId { get; set; } 
 
	[Range(1, int.MaxValue)]
	public int RapportPlanId { get; set; } 
 
	[Required] 
	[MaxLength(500)]
	public string RapportPlanValue { get; set; } 

	public vRapportPlan()
	{
    }

    }
}

File: redsheet-portal/src/app/database-models/v-rapport-plan.ts
import { required, maxLength, range } from '@rx/annotations';

export class vRapportPlan {
    constructor(vRapportPlan?: vRapportPlan )  {
        let properties = [ "cultureCountryId", "rapportPlanId", "rapportPlanValue",];
        for (let property of properties)
            if (vRapportPlan && vRapportPlan[property])
                this[property] = vRapportPlan[property];
    }
 
	cultureCountryId : number =   0 ;
 
    @range(1,2147483647)
	rapportPlanId : number =   undefined;
 
    @required()
    @maxLength(500)
	rapportPlanValue : string =   undefined;
}
